File: output/ChatGPT/codenet/C++/Java/s375006529.java
import java.util.Scanner;

public class s375006529 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String s = scanner.next();
        int n = s.length();
        int[] v = new int[26];
        for (int i = 0; i < n; i++) {
            v[s.charAt(i) - 'a']++;
        }
        String ans = "Yes";
        for (int i = 0; i < 26; i++) {
            if (v[i] % 2 != 0) {
                ans = "No";
                break;
            }
        }
        System.out.println(ans);
    }
}

File: output/ChatGPT/codenet/C++/Python/s375006529.py
s = input()
n = len(s)
v = [0] * 26

for i in range(n):
    v[ord(s[i]) - ord('a')] += 1

ans = "Yes"

for i in range(26):
    if v[i] % 2 != 0:
        ans = "No"
        break

print(ans)